applications/minotari_ledger_wallet/wallet/README.md

```diff
@@ -78,19 +78,18 @@ Ensure Docker Desktop is installed and running on your machine.
 
 The following command has to be run from the root of the Tari repository:
 
-```
-docker run --rm -it -v ".:/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder
-```
-
-This will load the Docker vm where you can now build the ledger app. Navigate to the wallet directory
-`applications/minotari_ledger_wallet/wallet` to build.
```

Comment:
Please retain this as an alternative

Reply:
Made further changes, please can you review again.